lua/core/autopairs.lua

```diff
@@ -1,51 +1,67 @@
 -- if not package.loaded['nvim-autopairs'] then
 --   return
 -- end
-local status_ok, autopairs = pcall(require, "nvim-autopairs")
-if not status_ok then
-  return
+local M = {}
+
+M.config = function()
+  O.plugin.autopairs = {
+    config = false,
+  }
 end
-local npairs = require "nvim-autopairs"
-local Rule = require "nvim-autopairs.rule"
 
--- skip it, if you use another global object
-_G.MUtils = {}
+M.setup = function()
+  if O.plugin.autopairs.config then
+    return O.plugin.autopairs.config
+  end
+
+  local status_ok, autopairs = pcall(require, "nvim-autopairs")
+  if not status_ok then
+    return
```

Comment:
Shouldn't this return `false`  ?

Reply:
I didn't change those lines.  They just got shifted down because I added code above it.  The return just makes sure the code below it stops executing.  There's nothing listening to the return value of the setup() function so you could return anything and it wouldn't change any behavior.

Reply:
Sure, I just pointed out that the return value of the function made no sens

Reply:
Tastyep, I don't know of any good place to put this, but I can tell by your pr's and this comment that you know your stuff.

Reply:
I changed it to return false